### ISSUE TITLE
dropout with manualSeed

### ISSUE THREAD

**Issue by cseeker** [torch/nn#1154]
Hi, I noticed that nn.dropout generate random mask without using  seed.
And, every learning trials, nn.dropout causes different results.
To prevent this issue, I want to make nn.dropout generate random mask with manualSeed.
Is there somebody who knows where I have to put 'torch.manualSeed(opt.seed)'  in nn.dropout?
Or is there any other way to generate random mask with manualSeed?

Thank you for reading

**Comment by achalddave** [comment]
I'm not sure what you mean. `nn.Dropout` does respect `torch.manualSeed`:
```
local nn = require 'nn'
local torch = require 'torch'

local dropout = nn.Dropout()

local input = torch.rand(5, 5)
torch.manualSeed(0)
local output1 = dropout:forward(input):clone()
torch.manualSeed(0)
local output2 = dropout:forward(input):clone()

print(torch.all(torch.eq(output1, output2))) -- true```
